i18n/en.json

```diff
@@ -0,0 +1,602 @@
+{
+  "%s day": {
+    "one": "%s day",
+    "other": "%s days"
+  },
+  "%s hour ago": {
+    "one": "%s hour ago",
+    "other": "%s hours ago"
+  },
+  "%s minute": {
+    "one": "%s minute",
+    "other": "%s minutes"
+  },
+  "%s minute ago": {
+    "one": "%s minute ago",
+    "other": "%s minutes ago"
+  },
+  "%s minute remaining": {
+    "one": "%s minute remaining",
+    "other": "%s minutes remaining"
+  },
+  "10th": "10th",
+  "11th": "11th",
+  "12th": "12th",
+  "13th": "13th",
+  "14th": "14th",
+  "15th": "15th",
+  "16th": "16th",
+  "17th": "17th",
+  "18th": "18th",
+  "19th": "19th",
+  "1st": "1st",
+  "20th": "20th",
+  "21st": "21st",
+  "22nd": "22nd",
+  "23rd": "23rd",
+  "24th": "24th",
+  "25th": "25th",
+  "2nd": "2nd",
+  "3rd": "3rd",
+  "4th": "4th",
+  "5th": "5th",
+  "6th": "6th",
+  "7th": "7th",
+  "8th": "8th",
+  "9th": "9th",
+  "add.failure.unrecognizedGroups.added": "added",
```

Comment:
Namespace all command translations with `command` or something similar

Reply:
General note: can't you namespace these by doing something like:
command: {
    help: {
        example: {
            exclude: "...",
            include: "..."
        }
    }
}